frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -2291,6 +2416,12 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
             if (device_type && device_type !== values.deviceTypeFilter) {
                 actions.setDeviceTypeFilter(device_type)
             }
+            if (tile_visualizations && !objectsEqual(tile_visualizations, values.tileVisualizations)) {
+                for (const [tileId, visualization] of Object.entries(tile_visualizations)) {
+                    // we only save graph visualizations since they are not the default option
+                    actions.setTileVisualization(tileId as TileId, visualization as 'graph')
+                }
+            }
```

Comment:
Not sure I follow this `graph` bit. On line 2303 you're storing `tileVisualizations` in its entirety on the query string, but here you're *assuming* it can only be `graph` which I'm not sure it's true.

I'm not against storing only the `graph` ones in the query, but you'll need to change the the `urlParams.set` call then